index.html

```diff
@@ -3514,6 +3514,58 @@ <h3>Reserved Extension Points</h3>
         </p>
 
       </section>
+
+      <section class="normative">
+        <h3>Ecosystem Compatibility</h3>
+
+        <p>
+There are a number of digital credential formats that do not natively use the
+data model provided in this document, but are aligned with a number of concepts
+in this specification. At the time of publication, examples of these digital
+credential formats include pure JSON Web Tokens (JWTs), CBOR Web Tokens (CWTs),
+ISO-18013-5 (mDLs), and Authentic Chained Data Containers (ACDCs).
+        </p>
+
+        <p>
+If conceptually aligned digital credential formats can be transformed to the
+data model according to the rules provided in this section, they are considered
+<em>"compatible with the Verifiable Credentials ecosystem"</em>, and the result
+of the transformation, which is a <a>conforming document</a>, is a
+<a>verifiable credential</a>. Specifications that contain transformations to the
+data model described by this specification:
```

Comment:
Two things:

1. Only the set of inputs for which a given transformation will work should be considered "compatible with the Verifiable Credentials ecosystem". While it's fine if someone wants to produce a one-off artisanal transformation of some particular expression of data in one format to the base media type, it's just that one example that would be "compatible". It would not somehow mean every instance of data in that format is now also "compatible". The number of inputs that could be considered "compatible" will only increase in a scalable way if these specs describe a generalized function that is capable of transforming as many expressions of data in some format to the base media type as possible. I think a slight tweak to the language here to talk about these specs defining the process of "how" a transformation is performed instead of just "containing 'transformations'" could help with that.
2. We need to frame this around transformation to the base media type, not the data model (just like another suggestion does above in some other area).

```suggestion
<a>verifiable credential</a>. Specifications that describe how to perform
transformations to the base media type (`application/vc+ld+json`):
```

Reply:
I have made an attempt at re-writing the language to be more consistent and use "conforming document" in commit 08ad96a0b98540c6b01257d10e1b3faf22c07fd9 and then be specific about what a "conforming document" is in bd9aab2a283bd45789984fe2fc528c696ebe2aac.